### PR TITLE
basic-features: updates for v6.6

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
basic-features: updates for v6.6

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,207 +20,207 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md)                       |   Y    |   Y     |   Y     |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md)       |   Y    |   Y     |   Y     |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [用户自定义变量](/user-defined-variables.md)                               | E | E | E | E | E | E | E | E | E |
+| [`BATCH [ON COLUMN] LIMIT INTEGER DELETE`](/sql-statements/sql-statement-batch.md) | Y | Y | Y | N | N | N | N | N | N |
+| [`BATCH [ON COLUMN] LIMIT INTEGER INSERT/UPDATE/REPLACE`](/sql-statements/sql-statement-batch.md) | Y | Y | N | N | N | N | N | N | N |
+| [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md) | Y | Y | E | N | N | N | N | N | N |
+| [表级锁 (Table Lock)](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入) | E | E | E | E | E | E | E | E | E |
+| [物化列式存储的查询结果](/tiflash/tiflash-results-materialization.md) | E | E | N | N | N | N | N | N | N |
```
Comment (songrijie) [inline basic-features.md:85]:
```suggestion
| [资源管控](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |

```
增加资源管控 resource control

File: basic-features.md [hunk 2]
```diff
@@ -20,207 +20,207 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Follower Read](/follower-read.md)                                  |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [Optimizer hints](/optimizer-hints.md)                              |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [MPP 执行引擎](/explain-mpp.md)                                     |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | N        |
+| [索引合并](/explain-index-merge.md)                                 |    Y    |    Y    |    Y    |    Y     | E | E | E | E | E |
+| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md)                |    Y    |    Y    |    Y    | E | E |    N     |    N     |    N     | N        |
+| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E |
```
Comment (songrijie) [inline basic-features.md:102]:
```suggestion
| [资源管控 (Resource Control)](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |

```
增加资源管控

File: basic-features.md [hunk 3]
```diff
@@ -20,207 +20,207 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DM WebUI](/dm/dm-webui-guide.md)                                         | E | E  | E |    N     |    N     |    N     |    N     |    N     | N        |
+| [前台限流](/tikv-configuration-file.md#前台限流)                            | Y | Y | E |    N     |    N     |    N     |    N     |    N     | N        |
+| [后台限流](/tikv-configuration-file.md#后台限流)                             | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore)                        |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/br-pitr-guide.md)   |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值)   |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | N |
```
Comment (songrijie) [inline basic-features.md:224]:
```suggestion
| [资源管控 (Resource Control) ](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |

```
增加资源管控

File: basic-features.md [hunk 4]
```diff
@@ -20,207 +20,207 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能)   |  E  |  E   |  E    |    E     |    E     |    E     |    E     |    E     |    E     |
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E | E | E | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md)               |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
+| [复合主键](/constraints.md#主键约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [唯一约束](/constraints.md#唯一约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [整型主键上的聚簇索引](/constraints.md)                                |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
```
Comment (songrijie) [inline basic-features.md:61]:
```suggestion
| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | N | N | N | N | N | N | N |

```
多值索引

File: basic-features.md [hunk 5]
```diff
@@ -20,207 +20,209 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [TiDB Dashboard Top SQL 功能](/dashboard/top-sql.md)                       |    Y    |    Y    |    Y     | E |    N     |    N     |    N     |    N     | N        |
+| [TiDB Dashboard SQL 诊断功能](/information-schema/information-schema-sql-diagnostics.md)    |    Y    |    Y    | E | E | E | E | E | E | E |
+| [TiDB Dashboard 集群诊断功能](/dashboard/dashboard-diagnostics-access.md)   |    Y    |    Y    | E | E | E | E | E | E | E |
+| [Grafana 中的 TiKV-FastTune 面板](/grafana-tikv-dashboard.md#tikv-fasttune-面板) |    E    |    E    |    E     |    E     |    E     |    E     |    E     |    E     | E        |
+| [Information schema](/information-schema/information-schema.md)           |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [Metrics schema](/metrics-schema.md)                                      |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [Statements summary tables](/statement-summary-tables.md)                 |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
```
Comment (mornyx) [inline basic-features.md:208]:
```suggestion
| [Statements summary tables](/statement-summary-tables.md)                 |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
| [Statements summary tables - 持久化](/statement-summary-tables.md#持久化-statements-summary)                 |    E    |    N    |    N     |    N     |    N     |    N     |    N     |    N     | N        |
```

File: basic-features.md [hunk 6]
```diff
@@ -20,207 +20,209 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能)   |  E  |  E   |  E    |    E     |    E     |    E     |    E     |    E     |    E     |
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E | E | E | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md)               |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
+| [复合主键](/constraints.md#主键约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [唯一约束](/constraints.md#唯一约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [整型主键上的聚簇索引](/constraints.md)                                |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | N | N | N | N | N | N | N |
```
Comment (easonn7) [inline basic-features.md:62]: | [外键约束](/constraints.md#foreign-key) | Y | N | N | N | N | N | N | N |

File: basic-features.md [hunk 7]
```diff
@@ -20,207 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [外键约束](/constraints.md#foreign-key) | Y | N | N | N | N | N | N | N |
```
Comment (TomShawn) [inline basic-features.md:62]:
```suggestion
| [外键约束](/constraints.md#外键约束) | Y | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 8]
```diff
@@ -20,207 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [前台限流](/tikv-configuration-file.md#前台限流)                            | Y | Y | E |    N     |    N     |    N     |    N     |    N     | N        |
+| [后台限流](/tikv-configuration-file.md#后台限流)                             | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore)                        |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [PITR](/br/br-pitr-guide.md)   |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值)   |    Y    |    Y    |    N    |    N     |    N     |    N     |    N     |    N     |  N        |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control) ](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |
```
Comment (TomShawn) [inline basic-features.md:229]:
```suggestion
| [资源管控 (Resource Control)](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 9]
```diff
@@ -20,207 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [下推计算结果缓存 (Coprocessor Cache)](/coprocessor-cache.md)       |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | E |
+| [Stale Read](/stale-read.md)                                        |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    N     | N        |
+| [Follower Read](/follower-read.md)                                  |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [Optimizer hints](/optimizer-hints.md)                              |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
+| [MPP 执行引擎](/explain-mpp.md)                                     |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | N        |
+| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) |  Y | N  | N | N | N | N | N | N | N |
+
```
Comment (TomShawn) [inline basic-features.md:102]:
```suggestion
```

File: basic-features.md [hunk 10]
```diff
@@ -20,207 +20,212 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`AUTO_INCREMENT` 列](/auto-increment.md)                    |    Y    |    Y     |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [`AUTO_RANDOM` 列](/auto-random.md)                          |   Y   |    Y   |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [TTL (Time to Live)](/time-to-live.md) |   E   |    E   |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |   Y   |    Y   |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| 在单条语句中添加多列                                         |  Y  |   Y  | E | E | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |   Y   |    Y   |    Y    |    Y     |    Y     |    Y     |    Y     |    N     |    N     |
+| [临时表](/temporary-tables.md)                               |   Y   |    Y   |    Y    |    Y     |    Y     |    N     |    N     |    N     |    N     |
+| [并行 DDL](/system-variables.md#tidb_enable_concurrent_ddl-从-v620-版本开始引入)  |   Y   |    Y   |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```
Comment (TomShawn) [inline basic-features.md:121]:
```suggestion
| 并行 DDL  |   Y   |    Y   |    N    |    N     |    N     |    N     |    N     |    N     |    N     |
```

**Pull request merged**